exercises/wordy/wordy_test.rb

```diff
@@ -3,95 +3,127 @@
 require 'minitest/autorun'
 require_relative 'wordy'
 
-class WordProblemTest < Minitest::Test
-  def test_add_1
-    assert_equal 2, WordProblem.new('What is 1 plus 1?').answer
+# Test data version:
+# aa12f2e
+class WordyTest < Minitest::Test
+  def test_addition
+    # skip
+    question = 'What is 1 plus 1?'
+    assert_equal 2, WordProblem.new(question).answer
   end
 
-  def test_add_2
+  def test_more_addition
     skip
-    assert_equal 55, WordProblem.new('What is 53 plus 2?').answer
+    question = 'What is 53 plus 2?'
+    assert_equal 55, WordProblem.new(question).answer
   end
 
-  def test_add_negative_numbers
+  def test_addition_with_negative_numbers
     skip
-    assert_equal(-11, WordProblem.new('What is -1 plus -10?').answer)
+    question = 'What is -1 plus -10?'
+    assert_equal(-11, WordProblem.new(question).answer)
```

Comment:
Hi there!

I originally thought that it was something enforced by rubocop, but I'm not sure now. Whenever I save the file without the parentheses, I get a warning in Vim `ambiguous first argument; put parentheses or a space even after `-' operator` (only for cases with a negative number).

I'm assuming that's maybe why the original cases also had parens around negative cases?